ts/features/wallet/component/card/WalletV2PreviewCards.tsx

```diff
@@ -3,86 +3,48 @@ import * as React from "react";
 import { connect } from "react-redux";
 import { Dispatch } from "redux";
 import { GlobalState } from "../../../../store/reducers/types";
-import {
-  bancomatListVisibleInWalletSelector,
-  bPayListVisibleInWalletSelector,
-  cobadgeListVisibleInWalletSelector,
-  privativeListVisibleInWalletSelector,
-  satispayListVisibleInWalletSelector
-} from "../../../../store/reducers/wallet/wallets";
+import { paymentMethodListVisibleInWalletSelector } from "../../../../store/reducers/wallet/wallets";
+import { PaymentMethod } from "../../../../types/pagopa";
 import BancomatWalletPreview from "../../bancomat/component/BancomatWalletPreview";
 import BPayWalletPreview from "../../bancomatpay/component/BPayWalletPreview";
 import CobadgeWalletPreview from "../../cobadge/component/CobadgeWalletPreview";
+import CreditCardWalletPreview from "../../creditCard/component/CreditCardWalletPreview";
 import PrivativeWalletPreview from "../../privative/component/PrivativeWalletPreview";
 import SatispayWalletPreview from "../../satispay/SatispayWalletPreview";
 
 type Props = ReturnType<typeof mapDispatchToProps> &
   ReturnType<typeof mapStateToProps>;
 
+const paymentMethodPreview = (pm: PaymentMethod): React.ReactElement | null => {
+  switch (pm.kind) {
+    case "Satispay":
+      return <SatispayWalletPreview key={pm.idWallet} satispay={pm} />;
+    case "Bancomat":
+      return <BancomatWalletPreview key={pm.idWallet} bancomat={pm} />;
+    case "CreditCard":
+      // We should distinguish between a plain credit card and a cobadge credit card.
+      // Unfortunately, the cobadge card doesn't have a own type but is a CreditCard that has an issuerAbiCode
+      return pm.info.issuerAbiCode !== undefined ? (
```

Comment:
Is this the same assertion we done here?
https://github.com/pagopa/io-app/commit/ee549cabecef18957f5cc4ec382c3a0146ba3cc4
If yes could it make sense to have a function that does the check?

Reply:
Yes of course, changed in 636c17f